packages/@lwc/babel-plugin-component/src/__tests__/fixtures/component/component-feature-flag/expected.js

```diff
@@ -0,0 +1,14 @@
+import _componentFeatureFlag from "@salesforce/featureFlag/TEST_FLAG";
+import _tmpl from "./test.html";
+import { LightningElement, registerComponent as _registerComponent } from "lwc";
+class Test extends LightningElement {
+  /*LWC compiler vX.X.X*/
+}
+const __lwc_component_class_internal = _registerComponent(Test, {
+  tmpl: _tmpl,
+  sel: "x-test",
+  apiVersion: 9999999,
+  componentFeatureFlag: Boolean(_componentFeatureFlag),
+  componentFeatureFlagModulePath: "@salesforce/featureFlag/TEST_FLAG"
```

Comment:
```suggestion
  componentFeatureFlag: {
    value: Boolean(_componentFeatureFlag),
    path: "@salesforce/featureFlag/TEST_FLAG"
  }
```
This feels cleaner, but it's probably not worth the perf overhead of creating an object...

Reply:
I changed it to an object. There won't be many components that use featureFlags, so I'm leaning towards keeping it clean.